kivy/core/text/markup.py

```diff
@@ -775,5 +816,10 @@ def p_restricted(line, uw, c):
         lw = sum([word.lw for word in line1])
         lh = max([word.lh for word in line1]) * line_height
         self.options = old_opts
-        return lw + 2 * xpad, lh + 2 * ypad, [LayoutLine(0, 0,
-            lw, lh, 1, 0, line1)]
+        if uw < sum([word.lw for word in line]):
```

Comment:
Ah, yes, I missed the `1`. It's done multiple times, because there are multiple `return` statements in the method. Maybe I should move it right before the returns and use a single variable + check if it's computed. Could be better this way?